cedar-policy-formatter/tests/snapshots/cedar_policy_formatter__pprint__fmt__tests__format_files@extended_has.cedar.snap

```diff
@@ -0,0 +1,28 @@
+---
+source: cedar-policy-formatter/src/pprint/fmt.rs
+expression: formatted
+input_file: cedar-policy-formatter/tests/extended_has.cedar
+---
+// An example from RFC
+permit (
+  principal is User,
+  action == Action::"preview",
+  resource == Movie::"Blockbuster"
+)
+when
+{
+  // extended has
+  principal
+  has
+    // contactInfo
+    contactInfo.
+      // address
+      address
+      .
+      // zip
+      zip &&
+  // we are safe to access all attributes
+  principal.contactInfo
+    .address
+    .zip == "90210"
+};
```

Comment:
This is not very pretty, should we open an issue to improve this formatting?  Or I guess the culprit is the comments; what does the formatter produce for this example without comments?

Reply:
I'll add another policy without comments.

Reply:
Will also open an issue after the PR is merged.